### PR TITLE
Add `kyselyCamelCaseHook`

### PR DESCRIPTION
**Pull request by alecmev**
Fixes https://github.com/kristiandupont/kanel/issues/547. I just started using Kysely (and Kanel) myself, so a second set of eyes would be welcome. I didn't build the docs because even if I pin `vitepress` to `1.0.0-rc.31` it still creates new assets, so I leave it to you 😉

### CODE REVIEW

File: packages/kanel-kysely/src/kyselyCamelCaseHook.test.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  expect(
+    [
+      ...output["foo"].declarations[0].properties,
+      ...output["foo"].declarations[1].properties,
+      ...output["bar"].declarations[0].properties,
+    ].map((x) => x.name),
+  ).toEqual([
+    "snakeCase",
+    "sCREAMINGSNAKECASE",
```
Comment (alecmev) [inline packages/kanel-kysely/src/kyselyCamelCaseHook.test.ts:89]: `screamingSnakeCase` would be preferred, probably. Is that something `@kristiandupont/recase` can do?

Comment (kristiandupont) [reply packages/kanel-kysely/src/kyselyCamelCaseHook.test.ts:89]: Hah no, I had never heard of that casing! But I can add it :-)

**Comment by kristiandupont** [comment]
Thank you for contributing, this is great. Before I merge it, one question: how common is this when using Kysely, do you know? Because an alternative might be to have this as an option in the original Kysely hook and not require two..

**Comment by alecmev** [comment]
The best I can do is this 😛 

[Repositories containing `new Kysely<`](https://github.com/search?q=%22new+Kysely%3C%22+-is%3Afork+path%3A*.ts&type=code): ~1200 matches  
[Repositories containing `kysely` and `new CamelCasePlugin`](https://github.com/search?q=%22kysely%22+%22new+CamelCasePlugin%22+-is%3Afork+path%3A*.ts&type=code): 184 matches

**Comment by kristiandupont** [comment]
Lol ok. Would you mind fixing the eslint error, then I'll merge and publish it

**Comment by kristiandupont** [comment]
This is now published. Thank you for contributing!

**Comment by alecmev** [comment]
Thank you! 👍

**Pull request merged**